backend/src/main/java/org/latexscribe/LatexScribe/domain/dto/TemplateDto.java

```diff
@@ -0,0 +1,18 @@
+package org.latexscribe.LatexScribe.domain.dto;
+
+
+import org.latexscribe.LatexScribe.domain.enums.TemplateCategory;
+import org.latexscribe.LatexScribe.domain.model.User;
+
+public record TemplateDto(
+        String name,
+
+        Long size,
+
+        byte[] content,
+
+        TemplateCategory category,
+
+        User user
```

Comment:
A dto should contain the user, also its missing some fields see  <https://sarahv.atlassian.net/jira/software/projects/KAN/pages>

Reply:
No need for the other fields, we dont need last modified since the teplates are gonna be static.  Nor tags since we have a category for them.

Reply:
Otherwise it will include all the fields in the user as well when we request from the controller.